### PR TITLE
Eliminating .space-left and the need for it. re: #287

### PR DESCRIPTION
**Pull request by webinista**
- Renames `cf-example-text` to `cf-txt-example`
- Uses complex selectors to eliminate need for `.space-left` class
- Changes button-related CSS to add a 20px right margin to all buttons, except when they're the last element child (typically a right-floated button).

### CODE REVIEW

**Comment by plusjeff** [comment]
For #287 (adding for a mention link)

**Comment by plusjeff** [comment]
@webinista Sorry :crying_cat_face:  -- merge conflict. Can you merge and then assign back to me?

**Comment by alex** [comment]
I think this accidentally included parts of a different, unrelated, PR?

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -256,3 +256,7 @@ $(function() {
         });
     }
 });
+
+$(document).on('ready', function() {
+    $('.cf-form-cond-req').trigger('input');
+})
```
Comment (alex) [inline app/assets/javascripts/application.js]: Is this related to this PR?

**Comment by alex** [comment]
Oop, another conflict here :-(

File: app/views/web/generate.html.erb [hunk 2]
```diff
@@ -17,8 +17,8 @@
     <input type="hidden" value="<%= @certification_date %>" name="certification_date">
 
     <button type="button" class="usa-button-gray cf-action-back cf-push-left">&laquo; Go back and make edits</button>
-    <a href="#confirm-cancel-certification" class="cf-action-openmodal space-left cf-btn-link">Cancel certification</a>
-    <button type="submit" class="cf-push-right">Upload and Certify</button>
+    <a href="#confirm-cancel-certification" class="cf-action-openmodal cf-btn-link">Cancel certification</a>
+    <button type="submit" class="btn btn-primary cf-push-right">Upload and Certify</button>
```
Comment (alex) [inline app/views/web/generate.html.erb]: `btn` accidentally got added back here.

**Comment by plusjeff** [comment]
Tests fine in IE11
